.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.rs]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+max_line_length = 90
```

Comment:
It should agree with rustfmt, though, [whose default `max_width` setting is 100](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#max_width).
On the other hand, it [uses different smaller widths for different expressions](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#Default%5C%20%5C(default%5C)%5C%3A). :confused:

Reply:
I think this is consistent — the `max_line_length` setting for editors will provide autowrapping longer lines or warn if the lines get beyond that length.  As such it works as a soft limit whereas we can see `rustfmt`'s requirements as a hard limit (since they'll be enforced by CI).  I based this off @ma2bd's editor setup so hopefully it's all congruous with the current codebase :)